package.json

```diff
@@ -8,12 +8,22 @@
     "request": "^2.88.0"
   },
   "devDependencies": {
-    "eslint": "^5.5.0",
-    "jest": "^23.5.0"
+    "ajv": "^6.5.2",
+    "eslint": "^5.6.0",
+    "eslint-config-tamia": "^6.0.1",
+    "eslint-plugin-prettier": "^2.6.2",
+    "jest": "^23.6.0",
+    "memfs": "2.9.4",
+    "prettier": "^1.14.2"
+  },
+  "engines": {
+    "node": ">=8.6"
   },
   "scripts": {
+    "pretest": "npm run lint",
```

Comment:
why do we need to run lint before testing? 

I don't agree.

Reply:
Fixed 4838ecc - I usually add linting to `pretest` for all of the reasons described [here](https://softwareengineering.stackexchange.com/questions/308515/whats-the-point-of-running-unit-tests-on-a-ci-server)

* The developer doesn't have the discipline to lint manually prior to push
* They have forgotten to lint or test locally on their PR

I notice that travis is **already** running the linter as a `script` so we're already protected from laziness I see, so you are right, we don't need to lint twice. This repo is a bit of an edge case though, as the linter is only really there for formatting the test harness, not the source code like a 'real' project.

This does mean however that when running locally devs should remember to the following:

```console
npm run lint && npm test
```
to ensure they are running the tests against well formatted code